vendor/yosensi/yo-meter-pulse.yaml

```diff
@@ -2,16 +2,16 @@ name: YO Meter Pulse
 description: The YO Meter Pulse input accepts connection to potential-free contacts. The device will not count pulses if the input is connected to contacts with any potential. Device input can detect pulses with different contact debounce times. This parameter is configurable and by default is 47 ms – up to max. 10 pulses per 1 s (boundary range 1-50 pulses per 1 s depends on the contact debouncing setting).
 # Hardware versions (optional, use when you have revisions)
 hardwareVersions:
-  - version: '2.0'
+  - version: "2.1"
```

Comment:
Also please run `make validate fmt`, our validation wants single quotes instead of double

Reply:
Alright will do!

Reply:
I added the old firmware versions back and added the new ones, restructured the files and also executed the 'make validate fmt' command.

Reply:
Thank you!